src/components/pagination/pagination.html

```diff
@@ -0,0 +1,26 @@
+<nav class="govuk-c-pagination" role="navigation" aria-label="Pagination">
+  <ul class="govuk-c-pagination__list">
+      <li class="govuk-c-pagination__item govuk-c-pagination__item--previous">
+        <a class="govuk-c-pagination__link" href="#" rel="prev">
+          <span class="govuk-c-pagination__link-title">
+            <svg class="govuk-c-pagination__link-icon" xmlns="http://www.w3.org/2000/svg" height="13" width="17" viewBox="0 0 17 13">
```

Comment:
It'd just mean updating the SVG would be a breaking change since GOV.UK Frontend can't hide the internals of it's markup.